app/src/main/java/fr/free/nrw/commons/mwapi/OkHttpJsonApiClient.java

```diff
@@ -242,7 +243,13 @@ public OkHttpJsonApiClient(OkHttpClient okHttpClient,
                     String label = binding.getSubclassLabel().getValue();
                     String entityId = binding.getSubclass().getValue();
                     entityId = entityId.substring(entityId.lastIndexOf("/") + 1);
-                    subItems.add(new DepictedItem(label, "", "", false,entityId ));
+                    String description = "";
+                    SubclassDescription subclassDescription = binding.getSubclassDescription();
+                    if (subclassDescription != null
+                        && subclassDescription.getXmlLang() != null) {
+                        description = subclassDescription.getValue();
+                    }
+                    subItems.add(new DepictedItem(label, description, "", false,entityId ));
```

Comment:
formatting a tiny bit off in this line, spaces after `,`, no space before `)`

Reply:
Hi @macgills 
Do you want me to change it to as below?
                    subItems.add(new DepictedItem(label,description,"",false,entityId));

Reply:
it was just this part `false,entityId )` that violated style. We want there to be spaces after `,`s.
` false, entityId)`

We also have our codestyle bundled with the project so if you were to highlight this section and press `ctrl+alt+l` it will do this for you. Unfortunately a lot of the project is not compliant with the style yet so you do have to go through the rigmarole of highlighting instead of just applying it to the entire file.